src/main/java/seedu/address/logic/commands/ViewCommand.java

```diff
@@ -32,7 +31,7 @@ public CommandResult execute(Model model) {
         requireNonNull(model);
         Project project = model.getFilteredProjectList().get(targetIndex.getZeroBased());
         model.setTargetProject(project);
-        UniqueStaffList list = project.getStaffList();
+        //UniqueStaffList list = project.getStaffList();
```

Comment:
It seems a bit unlikely to use this line, can remove.

Reply:
Sorry missed that, it has been removed.